linuxkpi/gplv2/include/linux/rcupdate.h

```diff
@@ -1,4 +1,13 @@
+#ifndef	__GPLV2_RCUPDATE_H__
+#define	__GPLV2_RCUPDATE_H__
+
 #include_next <linux/rcupdate.h>
 
-/* LKPI_TODO */
-#define	rcu_replace_pointer(rcu, ptr, c) 0
+#define	rcu_replace_pointer(rcu, ptr, c)				\
```

Comment:
This should be submitted for linuxkpi in base.

Reply:
https://reviews.freebsd.org/D30025